### ISSUE TITLE
Open a repo from url or user name and repo name

### ISSUE THREAD

**Issue by linsui** [git-touch/git-touch#205]
Is there any way to open a repo? It seems that I can only find repos by search. Thanks!

**Comment by pd4d10** [comment]
What would you suggest as a way to show this feature?

**Comment by linsui** [comment]
I thought that this feature can be merged in to the search box. When a string with a special format, e.g. git-touch/git-touch, is inputed, show a hint to open it directly.

**Comment by shreyas1599** [comment]
> I thought that this feature can be merged in to the search box. When a string with a special format, e.g. git-touch/git-touch, is inputed, show a hint to open it directly.

Do you mean opening the repo without waiting for search to load and then clicking on the search result?

**Comment by linsui** [comment]
Yep. And it's not only about waiting. It's difficult to find a repo by search if it's just a fork. I want a direct way to access a repo with link.
